package.json

```diff
@@ -55,6 +55,7 @@
   },
   "dependencies": {
     "@babel/parser": "7.16.8",
+    "@babel/plugin-syntax-typescript": "^7.21.4",
```

Comment:
There is no need to install this. This plugin is for babel itself, but we are only using the parser, which has no concept of plugin packages. For the parser plugins are simply strings that enable features.